settings.xml

```diff
@@ -1,112 +1,131 @@
+  
+  <!-- proxies
+     This is a list of proxies which can be used on this machine to connect to the network.
+     Unless otherwise specified (by system property or command-line switch), the first proxy
+     specification in this list marked as active will be used.If you are accessing network
+     via a proxy server, Un-comment the proxy options and fill in your proxy server detail.
+   -->
+  <proxies>
+    <!-- proxy
+      Specification for one proxy, to be used in connecting to the network.
+
+    <proxy>
+      <id>optional</id>
+      <active>true</active>
+      <protocol>http</protocol>
+      <username>proxyuser</</username>
+      <password>proxypass</password>
+      <host>proxy.host.net</host>
+      <port>80</port>
+      <nonProxyHosts>local.net|some.host.com</nonProxyHosts>
+    </proxy>
+    -->
+  </proxies>
+  
+  <profiles>
+
+    <!-- Configure the JBoss GA Maven repository -->
+    <profile>
+      <id>jboss-ga-repository</id>
+      <repositories>
+        <repository>
+          <id>jboss-ga-repository</id>
+          <url>http://maven.repository.redhat.com/techpreview/all</url>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+      <pluginRepositories>
+        <pluginRepository>
+          <id>jboss-ga-plugin-repository</id>
+          <url>http://maven.repository.redhat.com/techpreview/all</url>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+    <!-- Configure the JBoss Early Access Maven repository -->
+    <profile>
+      <id>jboss-earlyaccess-repository</id>
+      <repositories>
+        <repository>
+          <id>jboss-earlyaccess-repository</id>
+          <url>http://maven.repository.redhat.com/earlyaccess/all/</url>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+      <pluginRepositories>
+        <pluginRepository>
+          <id>jboss-earlyaccess-plugin-repository</id>
+          <url>http://maven.repository.redhat.com/earlyaccess/all/</url>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+
+    <profile>
+      <id>mead-jdg7</id>
+      <repositories>
+        <repository>
+          <id>mead-jdg7</id>
+          <name>MEAD JDG7 Maven Repository</name>
+          <url>http://download.lab.eng.bos.redhat.com/brewroot/repos/jb-dg-7-rhel-7-build/latest/maven/</url>
```

Comment:
Should we have this repo upstream?

Reply:
@diegolovison without that we receive a lot of errors on jenkins related with `org.infinispan:infinispan-bom` as you can see on Jenkins
[for example](https://jdg-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/jdg-func-quickstarts/QUICKSTART=rapid-stock-market,label_exp=OC/336/console)

Reply:
@diegolovison I forced the branch updated and changed the commit removing the settings.xml changes